<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeIngredientForm</class>
 <widget class="QDialog" name="RecipeIngredientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create a new Recipe Ingredient</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>You can choose either a raw ingredient or another existing recipe as the basis for your ingredient.</string>
       </property>
       <property name="text">
        <string>Base Ingredient</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="ingredients">
        <attribute name="title">
         <string>Ingredients</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListView" name="ingredientsView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="recipes">
        <attribute name="title">
         <string>Recipes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListView" name="recipesView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="countLabel">
         <property name="text">
          <string>Showing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="loadedCountIndicator">
         <property name="text">
          <string notr="true">0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="countLabel_2">
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalCountIndicator">
         <property name="text">
          <string notr="true">0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="loadMoreButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::ObjectRotateRight"/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="quantityLabel">
       <property name="text">
        <string>Quantity</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="quantityEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="unitLabel">
       <property name="text">
        <string>Unit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="unitList"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RecipeIngredientForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RecipeIngredientForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
